### PR TITLE
Issue #500 - Add hero CTA for API Docs on Getting Started

### PR DESCRIPTION
**Pull request by ALRubinger**

### CODE REVIEW

**Comment by ALRubinger** [comment]
Pending @csuwildcat @angiejones review and +1.

File: docs/docs-index.js [hunk 1]
```diff
@@ -27,6 +27,16 @@ function DocsIndex() {
           primary
         />
       </div>
+      <div className='padding-top--md'>
```
Comment (angiejones) [inline docs/docs-index.js:30]: The two big hero sections does not look nice. Let's brainstorm a better layout

**Review by angiejones — CHANGES_REQUESTED** [review]

File: docs/docs-index.js (hunk 1 above)
Comment (angiejones) [reply docs/docs-index.js:30]: Not sure you had the latest file, ALR. My formatting changes of the code image have been lost
https://github.com/TBD54566975/developer.tbd.website/pull/492/files

Comment (ALRubinger) [reply docs/docs-index.js]: Yep, I've just put back your image and image class changes on this branch to show:

```
          imgSrc="/img/docs-web5-code-connect.png"
          imgClass="w-[50%] bottom-[1vh]"
```

Looks a little wonky when it's in column view:

image

Anything you want to do I'm cool with if you want to push changes to fix it up however you think is best, then squash/merge this PR to put it to bed. :)

Comment (ALRubinger) [reply docs/docs-index.js]: Idea I took from a UX guideline thing where you have primary and secondary big CTAs, if possible for us:

3 column layout where Quickstart takes up 2 of the slots (66%) and Web5 API Reference takes up 1 (33%).

Comment (angiejones) [reply docs/docs-index.js]: oh i like that idea! and we don't need the code image there anymore since the card is much smaller

Comment (angiejones) [reply docs/docs-index.js]: a preview just to see if we like this

image

Comment (angiejones) [reply docs/docs-index.js]: here's 2-1 with code image
image

**Review by angiejones — APPROVED** [review]

**Comment by angiejones** [comment]
@ALRubinger good to merge

**Pull request merged**